golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -476,10 +493,17 @@ func Test_EndBlock_Events(t *testing.T) {
 	if len(msgsSent) != 1 {
 		t.Errorf("got msgs = %v, want one message", msgsSent)
 	}
-	gotMsg, err := decodeBalances([]byte(msgsSent[0]))
+	gotMsg, gotNonce, err := decodeBalances([]byte(msgsSent[0]))
 	if err != nil {
 		t.Errorf("decode balances error = %v", err)
```

Comment:
Make this `Fatalf` then - if we get here then `gotMsg` and `gotNonce` are likely garbage and we shouldn't compare further.

Reply:
Created and assigned #3425 to you.